crates/houston/src/error.rs

```diff
@@ -7,11 +7,15 @@ use std::io;
 pub enum HoustonProblem {
     /// ConfigDirNotFound occurs when the default OS config can't be found.
     #[error("Could not determine default OS config directory.")]
```

Comment:
```suggestion
    #[error("Could not determine default OS configuration directory.")]
```

(we use the word 'configuration' everywhere else in this error file)